demos/vc_issuer/src/main.rs

```diff
@@ -451,6 +451,21 @@ fn static_headers() -> Vec<HeaderField> {
     vec![("Access-Control-Allow-Origin".to_string(), "*".to_string())]
 }
 
+#[update]
+fn set_alternative_origins(alternative_origins: String) {
+    const ALTERNATIVE_ORIGINS_PATH: &str = "/.well-known/ii-alternative-origins";
+    ASSETS.with_borrow_mut(|assets| {
+        let asset = Asset {
+            url_path: ALTERNATIVE_ORIGINS_PATH.to_string(),
+            content: alternative_origins.as_bytes().to_vec(),
```

Comment:
Randomly landing here: you can parse the `String` with `serde_json` to validate the schema before creating the asset.

Reply:
Yes, but configuring invalid values is a feature that is useful when testing Internet Identity.

Reply:
But aren't you testing VC flow here, not II?

Anyway, like I said, was just a random thought, like maybe adding a mime type to the asset btw.